inf-cdk-python-aws/files/README.md

```diff
@@ -0,0 +1,62 @@
+# ODS AWS CDK Python Quickstarter
+
+This Quickstarter can be used to deploy AWS resources. Its primary usage is to build solutions based on AWS CDK Python.
+
+## What is the AWS CDK?
+
+The AWS Cloud Development Kit (CDK) is an open-source software development framework that allows you to define cloud infrastructure in code. It provides a high-level object-oriented abstraction to define AWS resources imperatively using programming languages like TypeScript, JavaScript, Python, Java, and C#.
+
+## AWS CDK with Python
+
+AWS CDK supports Python as one of the programming languages to define and manage cloud infrastructure. You can use Python to write your infrastructure code, which will be compiled into AWS CloudFormation templates. This allows you to leverage Python's features and libraries to create, modify, and manage your AWS resources.
+
+## How to get started?
+
+The Quickstarter will automatically create a new AWS CDK Python project in your component repository.
+In order to develop, the Python package installer, pip, and virtual environment manager, virtualenv, are required. The provided ODS agent comes already installed with these tools.
+
+If you want to develop locally, make sure those tools are available in your environment, too. To run your code locally execute the following commands:
+
+```
+$ python -mvenv .venv
```

Comment:
should this be aligned and use Makefile -> make init?

Reply:
interesting topic -  originally i proposed that but some colleagues do not like a make target approach ...